views/places.hbs

```diff
@@ -1,12 +1,12 @@
 
 <ul>
-  <h1 class="page-title">{{localText.placesTitle}}</h1>
+  <h1 class="page-title">{{localLang.placesTitle}}</h1>
   <div class="places_links">
-    <a href="../{{lang}}/add-place">{{localText.placesAdd}}</a><a href="../{{lang}}/edit-place">{{localText.placesEdit}}</a>
-
+    <a href="/{{../lang}}/add-place">Add Place</a><a href="{{../lang}}/edit-place">Edit Place</a>
```

Comment:
is this right? not using localLang?

Reply:
Yeah it was changed to localText in the middleware. Seemed less confusing than having locallLang and lang together.